docs/source/installation/osx.rst

```diff
@@ -61,9 +59,9 @@ but MacVim users need to install it using the following command::
 Fonts installation
 ==================
 
-Install downloaded patched font by double-clicking the font file in Finder, then 
-clicking :guilabel:`Install this font` in the preview window.
+Download the patched font from Github, install it by double-clicking the font file in Finder and then 
```

Comment:
And do not mention github here, it is not the *only* possible source of patched font downloads and it is absolutely irrelevant from where patched font was downloaded.